mrchecker-framework-modules/mrchecker-database-module/pom.xml

```diff
@@ -122,6 +122,17 @@
     <!-- Build goals/actions -->
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <db.connection.url>127.0.0.1:3306</db.connection.url>
+                        <db.username>root</db.username>
+                        <db.password>1234</db.password>
```

Comment:
Connection credentials (especially used only for testing purposes) shouldn't be present in pom.xml file